saved-sites/saved-sites-impl/src/main/res/layout/view_new_tab_favourites_section.xml

```diff
@@ -58,6 +57,7 @@
         tools:spanCount="4"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:paddingTop="@dimen/keyline_2"
```

Comment:
for clipToPadding to work we need to set the padding in this view, otherwise it won’t work.